libs/cosmos-sdk/x/auth/types/account.go

```diff
@@ -102,9 +102,7 @@ func (acc BaseAccount) Copy() sdk.Account {
 	return NewBaseAccount(acc.Address, acc.Coins, acc.PubKey, acc.AccountNumber, acc.Sequence)
 }
 
-var baseAccountBufferPool = amino.NewBufferPool()
-
-func (acc BaseAccount) AminoSize(cdc *amino.Codec) int {
+func (acc *BaseAccount) AminoSize(cdc *amino.Codec) int {
 	size := 0
 	if len(acc.Address) != 0 {
 		size += 1 + amino.ByteSliceSize(acc.Address)
```

Comment:
although it's not a modification this time,I'd appreciate it if a constant can be used instead of 1 :)